### ISSUE TITLE
Stellargraph install unnecessary

### ISSUE THREAD

**Issue by smithna** [deusebio/Graph-Machine-Learning#24]
Stellargraph is already in the requirements.txt and poetry.lock files.

**Comment by deusebio** [comment]
Yes, Indeed. I have just opened a PR to fix this. Thanks for flagging this!

**Comment by deusebio** [comment]
Closed by https://github.com/deusebio/Graph-Machine-Learning/pull/25
